pkg/networking/cilium/installation.go

```diff
@@ -25,6 +28,27 @@ type Installation struct {
 	ConfigMap *corev1.ConfigMap
 }
 
+// Version returns the version of the cilium Installation.
+func (i Installation) Version() (string, error) {
+	if i.DaemonSet == nil {
+		return "", nil
+	}
+
+	dsImage := i.DaemonSet.Spec.Template.Spec.Containers[0].Image
+	_, dsImageTag := oci.Split(dsImage)
```

Comment:
why not just return dsImageTag here?

Reply:
ah, yeah that makes things much simpler.